miqbox/miqbox.py

```diff
@@ -373,6 +373,30 @@ def start(connection, name):
     except Exception:
         click.echo("Fail to start appliance...")
 
+@cli.command(help="Restart Appliance Server")
+@click.argument("name")
+@connection
+def restart(connection, name):
+    """Stop running appliance
+
+    Args:
+        name: (`str`) appliance name
```

Comment:
It should be `str` or `id`. As appliance is running it will get some `id` and then user may want to restart `server`